config/locales/de.emails.yml

```diff
@@ -0,0 +1,14 @@
+# German translation for emails
+
+de:
+  emails:
+    send_email:
+      sending_successful: 'Email wurde erfolgreich versendet'
+    email_form:
+      to: 'An'
+      bcc: 'BCC'
```

Comment:
We dont need this one anymore (PO changed his opinion), so I suggest to remove it (and update the label on the buttons for (non) hiding the recipients)